src/hpr_routing.erl

```diff
@@ -88,16 +88,25 @@ deliver_packet(Packet, [Route | Routes]) ->
         "delivering packet to ~s",
         [hpr_route:lns(Route)]
     ),
-    %% FIXME: delivery could be halted to multiple routes if one of the earlier
-    %% Protocol:send(...) errors out.
+    hpr_packet_reporter:report_packet(Packet, Route),
```

Comment:
We're going to need to move this to make sure we don't account for packets that failed to be delivered.
@macpie I think you were thinking about where it could go?

Reply:
Yes we should move this to line 114 when we got an "ok" back from the protocol